root/mods-unpacked/otDan-BetterModList/translations/bettermodlist-translation.csv

```diff
@@ -1,5 +1,5 @@
-"key","en","it_IT","pl_PL"
-"MENU_MODS","Mods","Mods","Mody"
-"MOD_AUTHORS","Authors","Autori","Autorzy"
-"MOD_CONFIG","Config","Configurazione","Config"
-"MOD_DEPENDENCIES","Dependencies","Dipendenze","Zależności"
+"key","en","it_IT","pl_PL","uk"
```

Comment:
can you make sure the key is uk only? I believe it would be something like uk_UA at least or something along those lines

Reply:
Im sure

Reply:
hey forgot to merge this and so much time has passed already, could you check the translation file if you have some free time and add what's missing for the new things?

Reply:
Oke